src/install.rs

```diff
@@ -152,24 +154,27 @@ fn chmod_x(file: &PathBuf) {
 }
 
 fn mac_process_zip(archive_path: &PathBuf) {
-    // Using Mac unzip because it keeps the execution flags intact. 
+    // Using Mac unzip because it keeps the execution flags intact.
     let archive_parent = archive_path.parent().unwrap();
     let _output = Command::new("/usr/bin/unzip")
-                            .arg(archive_path.to_string_lossy().to_string())
-                            .arg("-d")
-                            .arg(archive_parent.to_string_lossy().to_string())
-                            .output()
-                            .expect("Problem in Mac postprocessing: Unzip failed");
+        .arg(archive_path.to_string_lossy().to_string())
+        .arg("-d")
+        .arg(archive_parent.to_string_lossy().to_string())
+        .output()
+        .expect("Problem in Mac postprocessing: Unzip failed");
```

Comment:
Missed one `expect` here. On second thought, this function should also return a `Result`, so if something goes wrong the calling function can abort.

Reply:
Code rewritten